.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -1,6 +0,0 @@
-<component name="InspectionProjectProfileManager">
-  <profile version="1.0">
-    <option name="myName" value="Project Default" />
-    <inspection_tool class="TypeAnnotation" enabled="false" level="WARNING" enabled_by_default="false" />
-  </profile>
-</component>
```

Comment:
Put this file back. IntelliJ automatically removes it when creating a new project for the first time. I don't at the moment know of a way to fix this, yet. For now: `git checkout develop -- .idea && git add .idea && git commit --amend -C HEAD`